### ISSUE TITLE
notice (undefined variable) in the_seo_framework_sitemap_exclude_ids filter

### ISSUE THREAD

**Issue by tescz** [sybrew/the-seo-framework#330]
Hello,
I got these errors when I using **the_seo_framework_sitemap_exclude_ids** filter.

**Notice:** Undefined variable: included in **/Users/Me/Documents/_www/htdocs_project/wordpress/wp-content/plugins/autodescription/inc/classes/sitemaps.class.php** on line **1075**
 
Solution is easy (I think), just add **$included** variable (with **false** value) to top of the is_post_included_in_sitemap() method (which is defined in /inc/classes/sitemaps.class.php - at line 1052). 

Thank you!! :)

**Comment by sybrew** [comment]
[This was reported on WordPress.org](https://wordpress.org/support/topic/how-to-exclude-one-id-from-sitemap/) earlier. Thanks for reminding me!

Here's a valid and sustainable workaround:
```php
add_filter( 'the_seo_framework_sitemap_exclude_ids', 'my_sitemap_exclude_ids' );
function my_sitemap_exclude_ids() {
    $ids = array( -1, 1086 );
 
    return $ids;
}
```

**Comment by tescz** [comment]
Oh, thanks for very fast reply and for the valid and sustainable workaround :). Have a nice day.

**Comment by sybrew** [comment]
Whoops... I got myself confused.

The issue I mentioned was resolved in 3.0.6.

The fix for this issue was brought in 6187fdd during sanity checks.
